src/Entity/Server.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+namespace Drupal\elastic_appsearch\Entity;
+use Drupal\Core\Entity\EntityStorageInterface;
+use Drupal\Core\Config\Entity\ConfigEntityBase;
+
+/**
+ * Defines the Server entity.
+ *
+ * @ConfigEntityType(
+ *   id = "server",
+ *   label = @Translation("Appsearch Server"),
```

Comment:
Same as above. `server` is a very generic term, It should be `elastic_appsearch_server`